Frontend/src/app/views/dashboard/projects/projects.component.scss

```diff
@@ -1,17 +1,14 @@
+@import "../../../../app/styles/display.scss";
+
 .container {
-  padding: 40px;
+  padding: 20px;
 
-  @media only screen and (max-width: 830px) {
-    padding-left: 10px;
-    padding-right: 10px;
+  @media only screen and (min-width: 660px) {
+      padding: 40px;
   }
 }

```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives

Reply:
Padding removed